homework/Repository.js

```diff
@@ -13,15 +13,36 @@ class Repository {
    * @param {HTMLElement} container The container element in which to render the repository.
    */
   render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+    const repositoryInformation = Util.createAndAppend('ul', container, {
+      id: 'repository-information',
+    });
+
+    const RepositoryName = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const description = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const forks = Util.createAndAppend('li', repositoryInformation, { text: '' });
+    const update = Util.createAndAppend('li', repositoryInformation, { text: '' });
+
+    RepositoryName.innerHTML = `Repository:<a target=_blank href='${this.repository.html_url}'> ${
+      this.repository.name
+    }</a>`;
+
+    description.innerHTML = `Description: ${this.repository.description}`;
+    forks.innerHTML = `Forks: ${this.repository.forks}`;
+    update.innerHTML = `Updated: ${this.repository.updated_at}`;
```

Comment:
Another thing here please - the date which is retrieved from the GitHub API comes in a `ISO 8601` format i.e `2018-04-06T17:18:06Z`. This can be shown in a more readable format like `dd/mm/yyyy`. For this, I would suggest that you explore the `Date` object of JavaScript and see if you can make this appear on UI in `dd/mm/yyyy` format - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
Done !